service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -270,25 +272,13 @@ void testCreateDatasetWithInvalidUser() {
   @Test
   void testCreateDatasetAdmin() throws JsonProcessingException {
     when(samService.hasGlobalAction(SamAction.CREATE_METADATA)).thenReturn(true);
-    when(datasetDao.create(new Dataset(SOURCE_ID, dataset.storageSystem(), METADATA)))
+    when(datasetDao.create(new Dataset(SOURCE_ID, dataset.storageSystem(), METADATA_AS_STRING)))
         .thenReturn(dataset);
     DatasetId id = datasetService.createDataset(dataset.storageSystem(), SOURCE_ID, METADATA);
     verify(jsonValidationService).validateMetadata(objectMapper.readTree(dataset.metadata()));
     assertThat(id, is(datasetId));
   }
 
-  @Test
-  void testCreateDatasetInvalidMetadata() {
```

Comment:
We don't need this test anymore, because typing will take care of this before it gets to this point now!